pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_guidance-on-using-xrefs.adoc

```diff
@@ -0,0 +1,55 @@
+[id="guidance-on-using-xrefs_{context}"]
+= Guidance on using xrefs
+
+[role="_abstract"]
+To ensure that internal cross-references, xrefs, work when you publish your content, use only inter-document xrefs. Although AsciiDoc supports different forms of xref, {ProductShortName} validates only inter-document xrefs. If validation fails, {ProductShortName} flags invalid xrefs.
+
+NOTE: {ProductShortName} only validates xrefs within a repository.
+
+The following are the syntaxes of inter-document xref:
+
+* `\xref:document-b.adoc#target-section[Target section]`
+* `\xref:../modules/document-c.adoc[Content C]`
+
+[dicsrete]
```

Comment:
s/dicsrete/discrete

alternatively, get rid of the h2 and use .Headings instead.

Reply:
fixed the spelling of discrete